test/e2e/create_cluster_test.go

```diff
@@ -50,8 +50,8 @@ func TestCreateClusterRequestServingIsolation(t *testing.T) {
 	ctx, cancel := context.WithCancel(testContext)
 	defer cancel()
 
-	nodePools := e2eutil.SetupRequestServingNodePools(ctx, t, globalOpts.ManagementParentKubeconfig, globalOpts.ManagementClusterNamespace, globalOpts.ManagementClusterName)
```

Comment:
:( I'm a fan of explicit naming to readers with less context don't have guess things